### ISSUE TITLE
Columns visibility configuration in results view

### ISSUE THREAD

**Issue by yanOnGithub** [dbeaver/dbeaver#3201]
like #3199

![20180324t113750-yani5](https://user-images.githubusercontent.com/25423046/37859995-f243b12a-2f57-11e8-85fe-92a334ecbe59.png)

**Comment by serge-rider** [comment]
I think this would be a good feature (set default column visibility in results view according to navigator filters).

Currently you can hide columns in results view thru context menu: 
![image](https://user-images.githubusercontent.com/6398845/38291801-fb75155a-37e9-11e8-91c2-a9be2dd189b1.png)

**Comment by serge-rider** [comment]
This sounds like two separate features:
1. Add global (on connection level) filter for columns
2. Use column filter settings in results viewer as default visibility config

**Comment by serge-rider** [comment]
![image](https://user-images.githubusercontent.com/6398845/38782229-030164d2-40f9-11e8-93e8-1e4d92275996.png)

**Comment by yanOnGithub** [comment]
I cannot show the 'LAST_%' columns. Did you hard code it?

**Comment by serge-rider** [comment]
I don't understand your use case.
In any case there is no any hardcode.

**Comment by yanOnGithub** [comment]
column missing in both v503 and v502

![image](https://user-images.githubusercontent.com/25423046/38795909-4e41a2d4-418c-11e8-902b-7fcb76104d11.png)

![image](https://user-images.githubusercontent.com/25423046/38795958-84741fda-418c-11e8-88ea-b2a5480df76d.png)

**Comment by serge-rider** [comment]
Can't reproduce that. DBeaver doesn't filter columns in results viewer at all (this feature isn't yet implemented).
Check your connection properties, perhaps these are two different databases.

**Comment by yanOnGithub** [comment]
The name1 varchar(256) column ONLY disappears on one of my computers. I cleared the dot dbeaver folder and the column failed to show. It worked on another computer.

I can hide the 'LAST_%' columns on the ER diagrams, but not the F4 table view.

**Comment by serge-rider** [comment]
Now I can't understand the actual problem.
Could you post steps to reduce this?

**Comment by yanOnGithub** [comment]
Sorry for the confusion. I have produced the following screenshots from the microsoft ie11 win7 x64 vm for vmware. This is supposed to be a clean reference environment available to both of us. I have tried both the connection-level column filter and table-level column filter. I cannot make the hello.name1 column disappear on the F4 table view.

https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

![20180421t074457-zoomit zoom window-yani5](https://user-images.githubusercontent.com/25423046/39077982-81b302d4-4538-11e8-85bf-e8cea8c4669b.png)
![20180421t074414-zoomit zoom window-yani5](https://user-images.githubusercontent.com/25423046/39077983-81dd553e-4538-11e8-9090-c55a27832c06.png)
